### PR TITLE
Update bicep references in recipes

### PR DESCRIPTION
**Pull request by sk593**

### CODE REVIEW

File: .github/scripts/validate-bicep.sh [hunk 1]
```diff
@@ -4,12 +4,13 @@ then
...
 fi
 
+WARNING_MSG="WARNING: The following experimental Bicep features"
 FILES=$(find . -type f -name "*.bicep")
 FAILURES=()
 for F in $FILES
 do
     echo "validating $F"
-    # We need to run the rad-bicep and fail in one of two cases:
+    # We need to run rad-bicep and fail in one of two cases:
```
Comment (lakshmimsft) [inline .github/scripts/validate-bicep.sh:13]: nit: bicep ?

File: bicepconfig.json
Comment (ytimocin) [inline bicepconfig.json:1]: This may need a reformat

**Review by ytimocin — APPROVED** [review]
Let's test it out 🚀

**Review by kachawla — APPROVED** [review]

File: .github/scripts/validate-bicep.sh [hunk 2]
```diff
@@ -4,12 +4,13 @@ then
     BICEP_EXECUTABLE="$BICEP_PATH/rad-bicep"
 fi
 
+WARNING_MSG="WARNING: The following experimental Bicep features"
```
Comment (kachawla) [inline .github/scripts/validate-bicep.sh:7]: Could you please add some context on what it is/how it is being consumed?

Comment (sk593) [reply .github/scripts/validate-bicep.sh:9]: Updated

File: bicepconfig.json
Comment (sk593) [reply bicepconfig.json:1]: Updated

**Review by kachawla — APPROVED** [review]

**Pull request merged**